charts/pipeline/values.yaml

```diff
@@ -196,6 +196,9 @@ configuration:
   #       defaultImages: {}
   #       defaultNetworkProvider: "cilium"
   #       defaultNodeVolumeSize: 0 # GiB, 0/fallback: max(50, AMISize)
+  #       disabled: false
+  #     azure:
+  #       disabled: false
```

Comment:
**Recommended**: We **SHOULD** use `enabled: true (default)` instead, because double negation increases cognitive complexity and enabled chart attributes are the norm throughout our Helm charts. These can be uncommented and left in to make sure the default is properly `enabled: true` without extra logic.

Reply:
Done! 🙂